docs/sc-tools/sc_machine.md

```diff
@@ -0,0 +1,33 @@
+# SC-machine
+
+!!! warning
+    This documentation is correct for only versions of sc-machine that >= 0.10.0.
+---
+
+SC-machine is a sc-memory with opportunity to add modules consistent with the general sc-machine interface. All such 
+modules are named extensions, must be shared libraries (with file extension `.so`) and stored in common directory 
+(by default it is `./bin/extensions`). By default, sc-machine contains sc-server that allows to communicate with 
+sc-memory through network. You can find `sc-machine` in a `bin` directory.
+
+There are options of sc-machine:
+
+<pre>
+./bin/sc-machine --help
+
+SC-machine usage:
+
+  --config|-c               Path to configuration file
+  --extensions_path|-e      Path to directory with sc-memory extensions
+  --repo_path|-r            Path to kb.bin folder
```

Comment:
Underscores in long CLI arguments are not a great idea as it's not standard / expected and can lead to various problems in visualization (in the docs and in the terminals)
[reference](https://stackoverflow.com/questions/1253679/should-command-line-options-in-posix-style-operating-systems-be-underscore-style)

Reply:
We will fix it latter. I created issue https://github.com/ostis-ai/sc-machine/issues/373